test/test_individual.py

```diff
@@ -56,6 +68,7 @@ def test_individual_with_parameter_python():
     assert y[0] == pytest.approx(x[0] + c)
 
 
+@pytest.mark.skipif(not torch_available, reason="torch is not installed.")
```

Comment:
for example in `test_cartesian_graph.py` we avoided the decorators and instead imported torch in the test functions like this `torch = pytest.importorskip("torch")`. is there a reason to prefer this over the other way? with `importorskip` we would avoid the `try...except` statement for the imports

Reply:
Yes, made the change, thanks.